frontend/src/scenes/ingestion/v1/panels/RecordingPanel.tsx

```diff
@@ -0,0 +1,42 @@
+import { SessionRecordingSettings } from 'scenes/session-recordings/settings/SessionRecordingSettings'
+import { useActions } from 'kea'
+import { ingestionLogic } from 'scenes/ingestion/v1/ingestionLogic'
+import { LemonButton, LemonDivider, Link } from '@posthog/lemon-ui'
+
+export function RecordingPanel(): JSX.Element {
+    const { setVerify } = useActions(ingestionLogic)
+
+    return (
+        <div style={{ maxWidth: 800 }}>
+            <h1 className="font-extrabold">Setup Session Recordings</h1>
+            <p className="prompt-text">
+                Session recordings allow you to see recordings of how your users are really using your product. It
+                includes powerful features like error tracking, filtering, and analytics to <b>diagnose UI issues</b>,{' '}
+                <b>improve support</b>, and <b>generally get inspired</b>.
+            </p>
+            <p>
+                No further configuration is required to immediately start capturing recordings. Learn more about
+                specific recordings settings in our{' '}
+                <Link to="https://posthog.com/manual/recordings#using-session-recording" target="_blank">
+                    docs
+                </Link>
+                .
+            </p>
+
+            <SessionRecordingSettings inModal />
+            <LemonDivider thick dashed className="my-6" />
+            <div>
+                <LemonButton
+                    type="primary"
+                    size="large"
+                    fullWidth
+                    center
+                    className="mb-2"
+                    onClick={() => setVerify(true)}
```

Comment:
We should only actually change the settings when the user clicks continue. If we think about it like a signup form, they can toggle their agreement to terms and conditions checkbox but the actual "agreement" takes place when they press continue.